frontend/src/components/ModelBox/ModelConfigurationSlider.tsx

```diff
@@ -32,23 +32,19 @@ function ModelConfigurationSlider({
 	const supportText = `more info about ${config.name}`;
 
 	return (
-		<div>
-			<div className="model-config-info">
-				<div className="model-config-title">
-					<p>{config.name}</p>
-				</div>
-				<button
-					className="model-config-info-icon prompt-injection-min-button"
-					title={supportText}
-					aria-label={supportText}
-					onClick={() => {
-						toggleInfo();
-					}}
-				>
-					<AiOutlineInfoCircle aria-hidden />
-				</button>
-			</div>
-			{showInfo && <div className="model-config-info-text">{config.info}</div>}
+		<fieldset className="model-config-slider-fieldset">
+			<legend>{config.name}</legend>
+			<button
+				className="info-icon prompt-injection-min-button"
+				title={supportText}
```

Comment:
I'd say remove the supportText const and change the title to "more info" as well? 
Interestingly, if the title and aria-label are different, they will both be read out, whereas when they are the same, at least NVDA doesn't repeat them, which is neat